cpp/src/search/contains.cu

```diff
@@ -14,22 +14,22 @@
  * limitations under the License.
  */
 
+#include <hash/unordered_multiset.cuh>
```

Comment:
Why was this moved here?

Reply:
This is moved up to before `#include <cudf/...` headers, as it is in the `src/` folder.